app/scripts/controllers/transactions/index.js

```diff
@@ -485,6 +491,8 @@ class TransactionController extends EventEmitter {
         txMeta.loadingDefaults = false
         this.txStateManager.updateTx(txMeta, 'transactions: gas estimation for tx on boot')
       }).catch((error) => {
+        tx.loadingDefaults = false
+        this.txStateManager.updateTx(tx)
```

Comment:
can you add a not for history in this update so we know that it was set false in the catch and didnt fail for another reason

Reply:
actually this is fine

Reply:
Done.